tests/Gelf/Test/LoggerTest.php

```diff
@@ -122,15 +123,38 @@ function (MessageInterface $message) use ($e, $line, $test) {
             );
         }
     }
+    
+    public function testLogMessage()
+    {
+        $test = $this;
+        $message = new Message();
+        $message->setLine(10)
+            ->setShortMessage("foo bar")
+            ->setAdditional("all", "nothing")
+            ->setAdditional("foo", "bar");
+        $additional = array(
+            "all" => "nothing",
+            "foo"=> "bar"
+        );
+        $this->validatePublish(
+            function (MessageInterface $message) use ($test, $additional) {
```

Comment:
You should be able to check for validity on object-level

    function (MessageInterface $publishedMessage) use ($message) {
        $this->assetzEquals($message, $publishedMessage);
    }